ru/mail/chrome/messenger/messengercompose/addressingWidgetOverlay.dtd

```diff
@@ -1,4 +1,4 @@
-<!-- address labels -->
+﻿<!-- address labels -->
```

Comment:
@yshchebet Let's not commit files that have no changes. It just creates more work to review.